src/main/scala/scala/tools/refactoring/analysis/IsNotInImports.scala

```diff
@@ -0,0 +1,53 @@
+package scala.tools.refactoring
+package analysis
+
+trait IsNotInImports { _: CompilationUnitDependencies with common.EnrichedTrees =>
```

Comment:
I think the problem lays in different place and I'm going to talk about it with Simon and Iulian. This problem is known as path dependent types. Because of it I can't right now rip out any reusable piece outside `OrganizeImports` class besides creating traits and mixing them with `OrganizeImports`. I really wish to have a bunch of fine grained classes/objects and compose them but with my current state of art it's not possible. Anyway it is not intended to be final, at least nowaday. I will work on it.